transaction-service/src/main/java/pl/adamboguszewski/transactionservice/application/TransactionPayment.java

```diff
@@ -1,5 +1,40 @@
 package pl.adamboguszewski.transactionservice.application;
 
 public class TransactionPayment {
-    // [TODO]: Implement this class.
+
+    public enum Currency {
+        PLN,
+        USD,
+        EUR
+    }
+
+    public enum PaymentType {
+        CASH,
+        CARD,
+        GOOGLE_PAY
+    }
+
+    private final Long amount;
+
+    private final Currency currency;
```

Comment:
I meant this to be amountPaid multiplier but this is ok, add Long multiplier then